code/script.js

```diff
@@ -1,47 +1,228 @@
 // All the DOM selectors stored as short variables
-const chat = document.getElementById('chat')
+const chat = document.getElementById("chat");
+const nameInput = document.getElementById("name-input");
+const submitBtn = document.getElementById("btn");
+const inputWrapper = document.getElementById("input-wrapper");
 
+//[updated] 'name' is reserved as global variable? instead 'name' -> 'username'
+let username;
 // Global variables, if you need any, declared here
 
 // Functions declared here
 
 // This function will add a chat bubble in the correct place based on who the sender is
 const showMessage = (message, sender) => {
-  if (sender === 'user') {
+  if (sender === "user") {
     chat.innerHTML += `
       <section class="user-msg">
         <div class="bubble user-bubble">
           <p>${message}</p>
         </div>
         <img src="assets/user.png" alt="User" />  
```

Comment:
I like that you changed the user image to have more of a brown background instead of blue, it looks nice with the rest of the styling! :)